packages/core-mobile/app/screens/portfolio/network/NetworkTokens.tsx

```diff
@@ -169,14 +169,26 @@ const NetworkTokens = (): JSX.Element => {
 
   const renderZeroState = (): JSX.Element => {
     return (
-      <View style={{ paddingHorizontal: 16, flex: 1, marginTop: -160 }}>
+      <View
+        style={{
+          paddingHorizontal: 16,
+          flex: 1,
+          marginTop: -160,
+          zIndex: -100
+        }}>
         <ZeroState.NetworkTokens goToReceive={goToReceive} />
       </View>
     )
   }
 
   const renderTokenTab = (): JSX.Element => {
-    if (tokenList.length === 0) return renderZeroState()
+    if (tokenList.length === 0)
```

Comment:
I feel like this showing manage button has been asked before. did you see a ticket for this?

Reply:
gotcha. let's leave it like that then.